### PR TITLE
enhancement: Token Period from env

### PR DESCRIPTION
**Pull request by juancaven1988**
# Description
Add a new env variable to define the token expiration date

# How Has This Been Tested?
Test funtions added in local

cc:

@dy46 @tommydangerous

### CODE REVIEW

File: mage_ai/settings/__init__.py [hunk 1]
```diff
@@ -27,10 +29,15 @@ def is_disable_pipeline_edit_access():
...
 
+#------------------------- DISABLE TERMINAL ----------------------#'
+
+DISABLE_TERMINAL = os.getenv('DISABLE_TERMINAL', '0').lower() in ('true', '1', 't')
+
 # ----------------- Authentication settings ----------------
 REQUIRE_USER_AUTHENTICATION = \
     os.getenv('REQUIRE_USER_AUTHENTICATION', 'False').lower() in ('true', '1', 't')
 AUTHENTICATION_MODE = os.getenv('AUTHENTICATION_MODE', 'LOCAL')
+TOKEN_PERIOD = os.getenv('TOKEN_PERIOD', '30d').lower()
```
Comment (dy46) [inline mage_ai/settings/__init__.py:40]: i think it would be better to make this a number value which represents the amount of time in seconds

Comment (dy46) [reply mage_ai/settings/__init__.py:40]: let's name it something a little more descriptive like: `MAGE_ACCESS_TOKEN_EXPIRY_TIME`

Comment (juancaven1988) [reply mage_ai/settings/__init__.py]: Add changes according to suggestions

File: mage_ai/settings/__init__.py [hunk 2]
```diff
@@ -27,10 +29,18 @@ def is_disable_pipeline_edit_access():
 
 DISABLE_TERMINAL = os.getenv('DISABLE_TERMINAL', '0').lower() in ('true', '1', 't')
 
+#------------------------- DISABLE TERMINAL ----------------------#'
+
+DISABLE_TERMINAL = os.getenv('DISABLE_TERMINAL', '0').lower() in ('true', '1', 't')
```
Comment (dy46) [inline mage_ai/settings/__init__.py:34]: it seems like there are some extra changes here that aren't related to the current issue. can you remove them?

**Comment by juancaven1988** [comment]
Add a new clean pull request

**Pull request closed without merging**